### PR TITLE
Add DOT payment

### PR DESCRIPTION
**Pull request by semuelle**

### CODE REVIEW

**Review by keeganquigley — DISMISSED** [review]
LGTM

**Review by Noc2 — APPROVED** [review]
Generally, it looks good. But maybe we can make the 30% DOT payment bold or something like this. I'm pretty sure nobody will read it ;-)

File: README.md [hunk 1]
```diff
@@ -141,7 +141,7 @@ The W3F Grants Program offers different grant levels to help you best, depending
...
+> **:loudspeaker:** Payments are made in DOT (30%) and USDT or USDC on the Polkadot [AssetHub](https://wiki.polkadot.network/docs/learn-assets) or fiat (USD, EUR, CHF). Please indicate your preference in the application [as outlined in our application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md?plain=1#L7). If you want to apply in **private**, you can do so [:arrow_right: here](https://docs.google.com/forms/d/e/1FAIpQLSfMfjiRmDQDRk-4OhNASM6BAKii7rz_B1jWtbCPkUh6N7M2ww/viewform). Note that this is generally a slower process and imposes stricter requirements on applicants.
```
Comment (takahser) [inline README.md:144]: I'd suggest to make it more explicit that the remaining 70% will be paid in stables or fiat.

File: docs/Process/how-to-apply.md [hunk 2]
```diff
@@ -9,7 +9,8 @@ If you want to apply in **private**, you can do so [:arrow_right: here](https://
 :::
 
 :::info
-Payments can be made in USDT and USDC on the Polkadot [AssetHub](https://wiki.polkadot.network/docs/learn-assets) and fiat (USD, EUR, CHF). Please indicate your preference in the application [as outlined in our application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md?plain=1#L7).
+
+Payments are made in DOT (30%, linearly vesting over 2 years) and USDT or USDC on the Polkadot [AssetHub](https://wiki.polkadot.network/docs/learn-assets) or fiat (USD, EUR, CHF). Please indicate your preference in the application [as outlined in our application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md?plain=1#L8).
```
Comment (takahser) [inline docs/Process/how-to-apply.md:13]: [Same](https://github.com/w3f/Grants-Program/pull/2224/files#r1496245303) here.

File: docs/faq.md [hunk 3]
```diff
@@ -15,6 +15,10 @@ title: 🙋 FAQ
 
   Generally, there is no upper limit to grant amounts. However, the higher the requested amount, the stricter the review. For guidance, please refer to the [grant levels section](Introduction/levels.md) in our documentation.
 
+### What does it mean for payments to be 30% DOT?
+
+  Every milestone payment will have two parts: 70% in the currency of your choosing (see the application template for options) and 30% in DOT, vesting linearly over two years. For this, we need a Polkadot AssetHub address in the grant agreement (if you haven't already provided it for USDC/USDT payments or you prefer a separate address) and, when it comes to milestone payments, two invoices: one for the currency part, one for the DOT part. Our [invoice form](https://docs.google.com/forms/d/18BdmYEt8gvcC-5f7TMHdJftBr9dEeA2_lUfIqkH7it8/viewform) will walk you through the process.
```
Comment (takahser) [inline docs/faq.md:20]:
```suggestion
  Every milestone payment will have two parts: 70% in the currency of your choice (see the application template for options) and the remaining 30% in DOT, vesting linearly over two years. For this, we need a Polkadot AssetHub address in the grant agreement (if you haven't already provided it for USDC/USDT payments or you prefer a separate address) and, when it comes to milestone payments, two invoices: one for the currency part, one for the DOT part. Our [invoice form](https://docs.google.com/forms/d/18BdmYEt8gvcC-5f7TMHdJftBr9dEeA2_lUfIqkH7it8/viewform) will walk you through the process.
```

**Review by takahser — COMMENTED** [review]
@semuelle mostly LGMT. Feel free to have a look at my inline comments, I think if we make it clear that it's a 30/70 split consistent throughout the docs we might be able to avoid some confusion.

File: README.md (hunk 1 above)
Comment (semuelle) [reply README.md:144]:
```suggestion
> **:loudspeaker:** Payments are made in DOT (30%) and _either_ USDT or USDC on the Polkadot [AssetHub](https://wiki.polkadot.network/docs/learn-assets) _or_ fiat (USD, EUR, CHF). Please indicate your preference in the application [as outlined in our application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md?plain=1#L7). If you want to apply in **private**, you can do so [:arrow_right: here](https://docs.google.com/forms/d/e/1FAIpQLSfMfjiRmDQDRk-4OhNASM6BAKii7rz_B1jWtbCPkUh6N7M2ww/viewform). Note that this is generally a slower process and imposes stricter requirements on applicants.
```

Comment (semuelle) [reply README.md:144]: Good?

File: docs/Process/how-to-apply.md (hunk 2 above)
Comment (semuelle) [reply docs/Process/how-to-apply.md:13]:
```suggestion
Payments are made in DOT (30%, linearly vesting over 2 years) and _either_ USDT or USDC on the Polkadot [AssetHub](https://wiki.polkadot.network/docs/learn-assets) _or_ fiat (USD, EUR, CHF). Please indicate your preference in the application [as outlined in our application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md?plain=1#L8). 
```

**Pull request merged**